src/data_snack/connections/memcached.py

```diff
@@ -21,5 +21,5 @@ def set_many(self, values: Dict[Text, Text]):
         failed_keys = self.connection.set_many(values)
```

Comment:
1. Yes, I will add it,
2. They are copied from the base class.
3. That would mean we need to add both of them to our requirements 
4. We don't want to use ABC here since non of those share any similarities in terms of the code base. The only think we care about is the interface of the functions, that's why we use Protocols here (in the same manner as we would use interfaces in java or c#).

Reply:
In a separate task